fhclient/main.go

```diff
@@ -25,6 +27,9 @@ func NewApiClient(c Config) ApiClient {
 
 	fhApiClient.transport = transport.New(c.ApiHost, "", []string{"https"})
 	fhApiClient.transport.Debug = c.Debug
+	if c.Transport != nil {
+		fhApiClient.transport.Transport = c.Transport
```

Comment:
Words cannot express how unhappy I am about this naming lol

Reply:
You are not alone -- this file needs some love.